app/views/pages/accessibility.html.erb

```diff
@@ -0,0 +1,48 @@
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <%= browser_back(@previous_step) %>
+
+    <h1 class="govuk-heading-xl">Accessibility statement</h1>
+
+    <p class="govuk-body">This accessibility statement applies to the Request personal information form at: <%= govuk_link_to "https://request-personal-info.form.service.justice.gov.uk" %>.</p>
+    <p class="govuk-body">This form was built by the Ministry of Justice. Justice Digital is responsible for the content of this form.</p>
+    <h2 class="govuk-heading-m">Using this form</h2>
+    <p class="govuk-body">We want as many people as possible to be able to use this form. For example, that means you should be able to:</p>
+    <ul class="govuk-list govuk-list--bullet">
```

Comment:
govuk-components has a method for lists - https://govuk-components.netlify.app/helpers/list/

Reply:
Thanks @vertism I have corrected that now